### DISCUSSION TITLE
Add feature to link private posts and anyway make them shareable by link

### DISCUSSION THREAD

**Discussion by rugk** [Traewelling/traewelling#2508] [Ideas / Feature Requests]
I would like to share a trip privately, i. E. not publicly, but anyway want to share a link for some user(s), where they can view my trip. 
Use case: e.g. if I want to meet with someone, I coukd send them a link and they can see in real time where I am. 

E.g. Travelynx has this feature already. There it is a visibility setting "linkable":

![Screenshot_20240420-142250_1](https://github.com/Traewelling/traewelling/assets/11966684/0bb5b3e0-6ab1-4ae1-b20a-b39d1bffd37a)

## implementation

IMHO it's does not need to be a visibility like in Travelynx, but coukd also be a button that generates a link (when the status is set to private). Though it may be hard to find out this feature exists then.

## related

See https://github.com/Traewelling/traewelling/discussions/2056 which might be related.

**Comment by timoschwarzer** [comment]
I'd guess the hardest part about this would be that status IDs are sequential integers instead of for example UUIDs. That would make an additional random URL parameter necessary for "unlisted" statuses.
